src/proxy.rs

```diff
@@ -122,6 +122,10 @@ impl<'a> Proxy<'a> {
         self.request(handle_token, method_name, body).await
     }
 
+    pub async fn version(&self) -> Result<u32, Error> {
+        self.property::<u32>("version").await
+    }
+
```

Comment:
This should be called once for the said proxy and stored in it struct so we don't have to fetch the value every time

Reply:
Done, should I transform the struct to
```rust
Proxy  {
    version: u32,
    inner: ...
}
```
too? or just `Proxy(x,y)` is enough?

Reply:
I would use what you suggested here to make it clearer.

Reply:
Would you mind documenting the version requirements? the next step, not necessarily to be done as part of this PR is to implement Deref for the inner proxy so we can make version available to all the portal objects.

Reply:
I have an idea how to do that, so don't bother with it :)

Reply:
Ok, I used the struct tho